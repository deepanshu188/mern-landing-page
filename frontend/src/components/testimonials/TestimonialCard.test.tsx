import { expect, test } from 'vitest';
import renderer from 'react-test-renderer';
import { TestimonailCard } from './TestimonialCard';

function toJson(component: renderer.ReactTestRenderer) {
  const result = component.toJSON();
  expect(result).toBeDefined();
  expect(result).not.toBeInstanceOf(Array);
  return result as renderer.ReactTestRendererJSON;
}

test('testimonial card', () => {
  const component = renderer.create(<TestimonailCard />);
  const tree = toJson(component);
  expect(tree).toMatchSnapshot();
});
