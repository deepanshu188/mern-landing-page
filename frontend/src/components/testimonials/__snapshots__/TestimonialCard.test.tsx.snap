// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`testimonial card 1`] = `
<div
  className="sc-bdfCDU bfTLgf"
>
  <div
    className="img-container"
  >
    <svg
      className="quote-icon"
      fill="none"
      height="22"
      viewBox="0 0 27 22"
      width="27"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M15.4 21.8V15C15.4 11.1067 16.3067 7.90667 18.12 5.4C19.9867 2.84 22.68 1.18666 26.2 0.439998V4.84C24.4933 5.26667 23.16 6.06667 22.2 7.24C21.24 8.36 20.6533 9.74667 20.44 11.4H24.6V21.8H15.4ZM0.600001 21.8V15C0.600001 11.1067 1.50667 7.90667 3.32 5.4C5.18667 2.84 7.88 1.18666 11.4 0.439998V4.84C9.69333 5.26667 8.36 6.06667 7.4 7.24C6.44 8.36 5.85333 9.74667 5.64 11.4H9.8V21.8H0.600001Z"
        fill="black"
      />
    </svg>
    <img
      src="/src/assets/img/testimonials/picture.png"
    />
  </div>
  <div
    className="right"
  >
    <div
      className="content"
    >
      <p
        className="thought"
      >
        “The results have been incredible. With Power Digital, it feels like they’re in our trench, supporting and understanding us. They’re like a partner and mentor in helping us get where we want to be.”
      </p>
      <p
        className="author"
      >
        <b
          className="name"
        >
          David Calathan -
        </b>
         Director of Design Operations, New York
      </p>
    </div>
    <div
      className="arrows"
    >
      <svg
        fill="none"
        height="22"
        viewBox="0 0 22 22"
        width="22"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M21 11H1"
          stroke="black"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
        <path
          d="M11 21L1 11L11 1"
          stroke="black"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
      <svg
        fill="none"
        height="22"
        viewBox="0 0 22 22"
        width="22"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M1 11H21"
          stroke="black"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
        <path
          d="M11 21L21 11L11 1"
          stroke="black"
          strokeLinecap="round"
          strokeLinejoin="round"
        />
      </svg>
    </div>
  </div>
</div>
`;
